### PR TITLE
added measure downtime script

### PR DESCRIPTION
**Pull request by pawelpaszki**
## What
added measure downtime script - https://issues.redhat.com/browse/INTLY-4985

## Verification
Executed here: https://master-jenkins-csb-intly.cloud.paas.psi.redhat.com/job/Playground/job/ppaszki-background-test/14/ (as a background process) and then gracefully stopped

If the artefact for some reason is no longer available for this build, see this json: https://gist.github.com/pawelpaszki/9c4a325b9853bfbb0211f385f56e59f3

### Note
I was intending to write this script in bash, but due to the complexity of the data structures used in the script and their manipulations while running the script, I stuck to JS.

### CODE REVIEW

**Comment by pawelpaszki** [comment]
ping @davidffrench @psturc @jhellar

File: scripts/ocm/measure-downtime.js [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+      writeJSONtoFile(); // write to file on each loop
+    }
+  }
+}
+
+/*
+Get all namespaces starting with the specified (or default) namespace prefix
+*/
+async function getProjects() {
+  console.log("Getting initial list of RHMI comopnents");
```
Comment (psturc) [inline scripts/ocm/measure-downtime.js]:
```suggestion
  console.log("Getting initial list of RHMI components");
```

**Comment by psturc** [comment]
The code LGTM @pawelpaszki. I will test it today when performing upgrade with latest build.

**Comment by pawelpaszki** [comment]
@psturc can anything be done about failing prow/format tests? I don't think node scripts will ever pass without changing the way this test runs

**Comment by psturc** [comment]
> @psturc can anything be done about failing prow/format tests? I don't think node scripts will ever pass without changing the way this test runs

I think you can change the path [here](https://github.com/integr8ly/rhmi-utils/blob/master/Makefile#L5) to filter out only `*.sh` files.

Would it make sense to add some `js` linter to our tests?

**Comment by pawelpaszki** [comment]
> > @psturc can anything be done about failing prow/format tests? I don't think node scripts will ever pass without changing the way this test runs
> 
> I think you can change the path [here](https://github.com/integr8ly/rhmi-utils/blob/master/Makefile#L5) to filter out only `*.sh` files.
> 
> Would it make sense to add some `js` linter to our tests?

If we end up developing more js tests than yes.

**Comment by jhellar** [comment]
LGTM, I think that it could be slightly refactored to avoid code duplication, but that could be also done later.

**Comment by pawelpaszki** [comment]
thanks for your reviews guys. Fixed one more typo and removed old comment. Can you add /approve and /lgtm please?

**Pull request merged**